### DISCUSSION TITLE
Dark Matter Reactor (idk if u do suggestions but its origins are from the this game https://www.roblox.com/games/3039795291/QS-Energy-Research-Facility#!/game-instances )

### DISCUSSION THREAD

**Discussion by PoliticalWater** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1249] [General]
yeah so a fusion reactor based on dark matter its inertial confinement fusion so it uses laser but has a metallic shell and boils water makes as much power as a balefire fusion reactor when at its average and meltdown would be a classic black hole then go boom that can be blocked only by the strongest blocks anyway it also uses gravity lasers and power lasers (it up to you have you want ur power lasers could be multi block) so in conclusion crappy fusionary watz with steam and dark matter and uses normal coolant (cold fusion) and lasers plus a metallic shell thingy def multi-block. (idk if u do suggestions but i just wanted to put this here anyway and you could put it into the mod).

**Comment by PoliticalWater** [comment]
plus idk if i put it in the right place thingy

**Reply by JameH2** [reply]
Im sorry but what the hell

**Reply by PoliticalWater** [reply]
its complicated..... wait are you the guy that made the fork of this mod?

**Reply by JameH2** [reply]
yeah
